### PR TITLE
Spell-check in OpenSUSE section of download page

### PR DESCRIPTION
**Pull request by wired-filipino-owl**
Changed URL so that Zypper is happy. Added an alias so that it's obvious that this repo is for Prism Launcher.

### CODE REVIEW

File: src/download/linux.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ RPM packages are available on the [Open Build Service](https://download.opensuse
   
 ```bash
 # add repository (if on leap, replace 'openSUSE_Tumbleweed' with '15.4')
-zypper addrepo https://download.opensuse.org/repositories/home:getchoo/openSUSE_Tumbleweed/home:getchoo.repo
```
Comment (getchoo) [inline src/download/linux.md:227]: the current instructions are from obs directly (see [here](https://software.opensuse.org//download.html?project=home%3Agetchoo&package=prismlauncher)), so the url shouldn't need to be changed afaik. an alias could also be nice, but i'm not sure if it's exactly the best choice here as technically my obs home repo is being added and not one specific to prism. if i were to ever move it over though, i'd be up for this

File: src/download/linux.md [hunk 2]
```diff
@@ -233,7 +233,7 @@ zypper install prismlauncher
 zypper install prismlauncher-nightly
 # stable releases (Qt5 version, available for Leap and Tumbleweed)
 zypper install prismlauncher-qt5
-# latest builds (avalible for Leap and Tumbleweed)
+# latest builds (available for Leap and Tumbleweed)
```
Comment (getchoo) [inline src/download/linux.md:236]: nice catch 😆

File: src/download/linux.md (hunk 1 above)
Comment (wired-filipino-owl) [reply src/download/linux.md:227]: Must have been some weird copy-paste error on my end.. I could've sworn `zypper` complained when I tried adding the repo when the URL had the .repo at the end.

**Comment by wired-filipino-owl** [comment]
Maybe a whole PR is a bit much just for spell-check, but I already put the time in haha

File: src/download/linux.md (hunk 1 above)
Comment (getchoo) [reply src/download/linux.md]: it will if you include the repo name iirc, but otherwise it's fine

**Review by getchoo — APPROVED** [review]

**Comment by getchoo** [comment]
> Maybe a whole PR is a bit much just for spell-check, but I already put the time in haha

hey, every bit helps :3

**Review by Scrumplex — APPROVED** [review]

**Pull request merged**